OrbitClientGgp/ClientGgp.cpp

```diff
@@ -81,7 +82,6 @@ bool ClientGgp::StopCapture() {
 }
 
 void ClientGgp::InitCapture() {
-  Capture::Init();
   target_process_ = GetOrbitProcessByPid(options_.capture_pid);
   CHECK(target_process_ != nullptr);
   // TODO: remove this line when GTargetProcess is deprecated in Capture
```

Comment:
```suggestion
```

Reply:
As Capture::SetTargetProcess(target_process_); is not needed to be removed in the end this comment is not necessary anymore.

Reply:
Well `Capture::SetTargetProcess(target_process_);` will be gone completely (hopefully by the end of tomorrow ;)).
As you do not rely on GSamplingProfiler, I will remove the comment and also the call to SetTargetProcess.

Reply:
Ok, perfect then, thanks!